src/ui/zcl_abapgit_gui_chunk_lib.clas.abap

```diff
@@ -218,12 +209,21 @@ CLASS zcl_abapgit_gui_chunk_lib IMPLEMENTATION.
 
     ro_html->add( |<div class="float-right message-panel-commands">| ).
 
+    lv_msgid_and_msgno = |{ ix_error->if_t100_message~t100key-msgid }{ ix_error->if_t100_message~t100key-msgno }|.
+
     IF lv_msgid_and_msgno IS NOT INITIAL.
+
+      lv_title = get_t100_text(
+                    iv_msgid = ix_error->if_t100_message~t100key-msgid
+                    iv_msgno = ix_error->if_t100_message~t100key-msgno ).
+
       ro_html->add_a(
-          iv_txt = lv_msgid_and_msgno
-          iv_typ = zif_abapgit_html=>c_action_type-sapevent
-          iv_act = zif_abapgit_definitions=>c_action-goto_message
-          iv_id  = `a_goto_message` ).
+          iv_txt   = lv_msgid_and_msgno
```

Comment:
What about changing this to just 'Message' or 'Message ({msgid}/{msgno})' ? The msg number does not say anything to a user. And those who are interested will click and see all the details. Imho just 'Message' is enough and it's number can be concatenated into title.

Reply:
![image](https://user-images.githubusercontent.com/17437789/62125698-25e0b000-b2ce-11e9-967b-d3f0c68c9252.png)

Reply:
Looks fine :)